libs/domain/user/registration/registration.component.ts

```diff
@@ -0,0 +1,260 @@
+          style="--column-gap: 20px;--row-gap: 15px;"
+        >
+          ${this.fieldRenderer.buildForm(this.getFields())}
+
+          <oryx-button
+            .size=${ButtonSize.Md}
+            ?loading=${this.isLoading}
+            @click=${this.onSubmit}
+          >
+            ${this.i18n('user.registration.sign-up')}
+          </oryx-button>
+        </oryx-layout>
+      </form>
+    `;
+  }
+
+  protected getFields(): FormFieldDefinition[] {
+    const passwordAttributes: FormFieldAttributes = {
+      minLength: this.$options()?.minLength as number,
+      maxLength: this.$options()?.maxLength as number,
+      minUppercaseChars: this.$options()?.minUppercaseChars as number,
+      minNumbers: this.$options()?.minNumbers as number,
+      minSpecialChars: this.$options()?.minSpecialChars as number,
+      strategy: this.$options()?.passwordVisibility as string,
+      errorMessage: (this.hasPasswordError &&
+        this.i18n(
+          'user.registration.password-doesn’t-satisfy-all-the-criteria'
+        )) as string,
+    };
+
+    const cleanedPasswordAttributes: FormFieldAttributes = Object.fromEntries(
+      Object.entries(passwordAttributes).filter(
+        ([key, value]) => value !== undefined
+      )
+    );
+
+    return [
+      {
+        id: 'salutation',
+        type: FormFieldType.Select,
+        required: true,
+        label: this.i18n('user.registration.title'),
+        options: this.salutations().map((option) => ({
+          ...option,
+          text: this.i18n(`user.registration.title.${option.text}`),
+        })),
+      },
+      {
+        id: 'gender',
+        type: FormFieldType.Select,
+        required: true,
+        label: this.i18n('user.registration.gender'),
+        options: this.genders().map((option) => ({
+          ...option,
+          text: this.i18n(`user.registration.title.${option.text}`),
+        })),
+      },
+      {
+        id: 'firstName',
+        type: FormFieldType.Text,
+        required: true,
+        label: this.i18n('user.registration.first-name'),
+      },
+      {
+        id: 'lastName',
+        type: FormFieldType.Text,
+        required: true,
+        label: this.i18n('user.registration.last-name'),
+      },
+      {
+        id: 'email',
+        type: FormFieldType.Email,
+        required: true,
+        label: this.i18n('user.registration.email'),
+        width: 100,
+      },
+      {
+        id: 'password',
+        type: FormFieldType.Password,
+        required: true,
+        label: this.i18n('user.registration.password'),
+        attributes: cleanedPasswordAttributes,
+        width: 100,
+      },
+      {
+        id: 'acceptedTerms',
+        type: FormFieldType.Boolean,
+        required: true,
+        label: this.i18n('user.registration.i-accept-the-<terms>', {
+          terms: {
+            value: 'terms and conditions',
+            link: {
+              href: this.$options()?.termsAndConditionsLink || '',
```

Comment:
Let's align the link and text of the link with the t&a from the footer. 
we should probably add a default value for the option here.